man/create_SparseM_reduction_matrix.Rd

```diff
@@ -0,0 +1,19 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/utils.R
+\name{create_SparseM_reduction_matrix}
+\alias{create_SparseM_reduction_matrix}
+\title{Helper function to slm_fit_csr}
+\usage{
+create_SparseM_reduction_matrix(zeroes)
+}
+\arguments{
+\item{x}{logical vector indicating which entries of x'x are zeroes.}
```

Comment:
This `\item{x}` should read `\item{zeroes}`, I'm pretty sure.  As it does in the roxygen source -- you probably just need to run `devtools::document()` and commit the result. As far as I can tell, this seems to be the source of the check failures being reported by the continuous integration testing.